pkg/controller/add_controllers.go

```diff
@@ -0,0 +1,11 @@
+package controller
```

Comment:
Whenever a new controller is added a new file called `add_$controller_name.go` is added to `pkg/controllers`. Since there is new file for every controller which is kind of too much I decided to aggregate them into a single file called `add_controllers.go`. So this is not part of the initial generated operator skeleton.

Reply:
Ack.